src/workshops/4/pgr.c

```diff
@@ -0,0 +1,53 @@
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+enum { TWO = 2, FIVE = 5, NINE = 9 };
+
+/* Denne funktion skal returnere 1 hvis x er et primtal og 0 ellers */
+int isPrime(int number) {
+  int result = 1;
+  for (int i = 2; i < number / 2; i++) {
+    if (number % i == 0) {
+      result = 0;
+      break;
+    }
+  }
+  return result;
+}
+
+/* Denne funktion skal returnere 1 hvis gcd(x,2)=1 og 0 ellers */
+int isGcd1(int number) { return number; }
+
+/* Denne funktion skal returnere 1 hvis 9^x-2 mod 5 = 2 og 0 ellers */
+int is2mod5(int number) {
+  int result = ((int)pow(NINE, number % FIVE) - TWO) % FIVE == TWO ? 1 : 0;
+  return result;
+}
+
+int main(void) {
+  int number_to_test = 0;
+  int isPrime_bool = 0;
+  int isGcd1_bool = 0;
+  int is2mod5_bool = 0;
```

Comment:
@MadsHeilmann Very strange variable names which don't explain the intention very well. Try something like
```suggestion
  bool is_prime_number = false;
  bool is_greatest_common_divisor = false;
```

Reply:
Naming is updated. ClangTidy wants to initilize to "0" and not "false" so i have kept is as "0".